change/@fluentui-react-toolbar-5c2c0e49-70bf-4b6d-b5e0-7a1dc8852ba9.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 6 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27797&commitId=239c3ee64248199f31212b0a5a6b538f176f6849&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1927 screenshots unchanged, 0 screenshots with different dimensions and 6 screenshots with visible difference.


  <details><summary><b>unknown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toolbar Converged.Large.Button Pressed.chromium.png|150| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|165| Changed|
|Toolbar Converged.Large.default.chromium.png|165| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|111| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|120| Changed|
|Toolbar Converged.Small.default.chromium.png|120| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>